openapi/data_repository_service.swagger.yaml

```diff
@@ -350,7 +352,9 @@ definitions:
           A name declared by the bundle author that must be
           used when materialising this object, 
           overriding any name directly associated with the object itself.
-          This string MUST NOT contain any slashes.
+          If this is a content object inside of a bundle then the name must be unique within that bundle. 
+          This string is made up of uppercase and lowercase letters, decimal digits, hypen, period, and underscore [A-Za-z0-9.-_]. See https://tools.ietf.org/html/rfc3986#section-2.3[RFC 3986 § 2.3].
```

Comment:
Is there a reason that we want names to match URIs? Maybe cite [POSIX filenames](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_282) instead? (I think it's the same characters, but makes more sense as a reference.)